Mathlib/RingTheory/Coalgebra.lean

```diff
@@ -66,3 +66,8 @@ instance Finsupp.instCoalgebra (R : Type u) (S : Type v) [CommRing R] : Coalgebr
     ext; simp
   id_ε := by
     ext; simp
+
+theorem Finsupp.Δ_basis (R : Type u) (S : Type v) [CommRing R] (s : S) :
```

Comment:
Following the [naming guide](https://leanprover-community.github.io/contribute/naming.html), we usually name things quite literally whenever we can.
```suggestion
theorem Finsupp.Δ_single_one (R : Type u) (S : Type v) [CommRing R] (s : S) :
```
I also recommend putting these in a `section` (or better yet, `namespace Finsupp`, and hoisting `R` and `S`  to a `variable` line at the start of the section.